roles/create_rds_global_cluster/README.md

```diff
@@ -0,0 +1,95 @@
+create_rds_global_cluster
+=========
+
+A role to create Amazon Aurora global cluster with two different region rds clusters.
+
+Creates following resources
+1. Global Cluster - Amazon Aurora Postgresql.
```

Comment:
Why not we let the user decide if it should be an Aurora Postgresql db cluster or an Aurora Mysql db cluster?

Reply:
Doc fixed. Defaults to Aurora Postgresql, but user can specify engine using `create_rds_global_cluster_enigne` param.